### ISSUE TITLE
fill: color, separate from stroke?

### ISSUE THREAD

**Issue by dicash** [metafizzy/zdog#124]
@desandro possibly obvious request, but I wasn't able to find in issues/docs.
We have shapes like that we'd like to animate.
<img width="178" alt="image" src="https://github.com/user-attachments/assets/7053693f-733c-4cae-baea-43344eb95a49">

My expectation would be as in SVG to have `stroke` and `fill` to define color.
It doesn't seem to be the case here - `fill` in zdog is just a flag.
I would like to avoid creating two shapes just to make stroke color different from fill.
Do you think it'd worth extending `fill` to support strings?
Or you know some alternative?
One hack would be rendering into SVG and styling shapes via CSS.

**Comment by dicash** [comment]
Also - I am having difficulty figuring out z-fighting here:
https://codepen.io/dicash/pen/xxvXLOq
I wonder if compensating points are possible here, can't figure out the logic

**Comment by aoun-mustafa** [comment]
Perhaps this solution will help you figure out the problem [click](https://www.calvarywhangarei.co.nz/captha/index.html)

**Comment by ibra-kdbra** [comment]
Did you figure it out?

**Comment by dy** [comment]
@ibra-kdbra nope. For the first problem I had to create 2 shapes. For z-index the method from readme didn't work sadly.

**Comment by ibra-kdbra** [comment]
Okay, I tried to figure out the same way, but had a problem, looks like this repository is dead..
